language/stdlib/modules/Libra.move

```diff
@@ -262,6 +262,20 @@ module Libra {
         };
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Treasury Compliant specific methods for DDs
+    ///////////////////////////////////////////////////////////////////////////
+
+    // Publish `preburn` under 'account'. Used for bootstrapping designated dealer
+    // as assocation TC account is creating this resource for DD
+    public fun publish_preburn_to_account<Token>(creator: &signer, account: &signer) {
+        Association::assert_account_is_blessed(creator);
+        let preburn = Preburn<Token> { requests: Vector::empty(), is_approved: true };
```

Comment:
Now that we have `move_to` and this awesome logic, do we need the `is_approved` flag in the preburn resource anymore?

Reply:
I was going to deprecate it in a followup PR since there are a whole bunch of uses of `new_preburn` is used in a whole bunch of places